app/products/playbooks/screens/playbooks_runs/playbook_card/playbook_card.tsx

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {defineMessage, useIntl} from 'react-intl';
+import {Text, TouchableOpacity, View} from 'react-native';
+
+import {CHIP_HEIGHT} from '@components/chips/constants';
+import UserChip from '@components/chips/user_chip';
+import FriendlyDate from '@components/friendly_date';
+import UserAvatarsStack from '@components/user_avatars_stack';
+import {useTheme} from '@context/theme';
+import ProgressBar from '@playbooks/components/progress_bar';
+import {goToPlaybookRun} from '@playbooks/screens/navigation';
+import {openUserProfileModal} from '@screens/navigation';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+import type PlaybookRunModel from '@playbooks/types/database/models/playbook_run';
+import type UserModel from '@typings/database/models/servers/user';
+import type {AvailableScreens} from '@typings/screens/navigation';
+
+const VERTICAL_PADDING = 16;
+const TITLE_HEIGHT = 24; // From typography at 200 size
+const GAP = 8;
+const GAPS = GAP * 2;
+export const ITEM_HEIGHT = (VERTICAL_PADDING * 2) + TITLE_HEIGHT + GAPS + (CHIP_HEIGHT * 2);
+
+const getStyleFromTheme = makeStyleSheetFromTheme((theme) => ({
+    cardContainer: {
+        borderRadius: 4,
+        backgroundColor: theme.centerChannelBg,
+        borderWidth: 1,
+        borderColor: changeOpacity(theme.centerChannelColor, 0.08),
+        shadowColor: '#000',
+        shadowOffset: {width: 0, height: 2},
+        shadowOpacity: 0.08,
+        shadowRadius: 3,
+        elevation: 2,
+    },
+    contentContainer: {
+        paddingVertical: VERTICAL_PADDING,
+        paddingHorizontal: 20,
+        gap: GAP,
+        flexDirection: 'column',
+    },
+    cardTitle: {
+        ...typography('Body', 200, 'SemiBold'),
+        color: theme.centerChannelColor,
+    },
+    peopleRow: {
+        flexDirection: 'row',
+        alignItems: 'center',
+        gap: 4,
+    },
+    infoRow: {
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+    },
+    lastUpdatedText: {
+        color: changeOpacity(theme.centerChannelColor, 0.64),
+        ...typography('Body', 75, 'Regular'),
+    },
+    flex: {
+        flex: 1,
+    },
+}));
+
+type Props = {
+    run: PlaybookRunModel | PlaybookRun;
```

Comment:
shouldn't this be a simple type instead of both?

I've only seen it used in app/products/playbooks/screens/playbooks_runs/playbook_runs.tsx (I might be wrong since github is having trouble with this PR size), and there is a list of models. Are there any other uses, if not, we should remove PlaybookRun from the types and fix the tests (which might be the ones causing the trouble with types)

Reply:
We don't store all the runs in the database. We only store all active runs, and up to 5 finished runs. The rest will come from the server. Those runs that come from the server also need to be renderable, and they come in the form of "API PlaybookRun". That is why we support both.